app/models/team.rb

```diff
@@ -158,8 +158,17 @@ def name
     UserGroup.find_by(metadata_id: GroupsMetadataCouncils.find_by(friendly_id: self.friendly_id)&.id, metadata_type: 'GroupsMetadataCouncils')
   end
 
+  private def team_if_any
+    UserGroup.find_by(metadata_id: GroupsMetadataTeamsCommittees.find_by(friendly_id: self.friendly_id)&.id, metadata_type: 'GroupsMetadataTeamsCommittees')
```

Comment:
Can you lookup the metadata entity and then just have a Rails relation? Like `has_one` in the metadata... This query is definitely too complicated!

Reply:
Oh I didn't think of that, did now. This is more easier. Did now. I might be able to simplify it further by creating a relation between team and groups_metadata_teams_committees through friendly_id, but didn't do because I anyway have to remove this after few PRs.